app/scripts/services/dimStoreService.factory.js

```diff
@@ -540,6 +540,12 @@
           index = index + item.itemInstanceId;
         }
 
+        /* 0: titan, 1: hunter, 2: warlock, 3: any */
+        var ArtifactTypes = ["Titan Artifact","Hunter Artifact","Warlock Artifact"]
```

Comment:
To fit with the existing code, please add a semicolon and add commas after each array element. You probably also want to name the variable `artifactTypes` to fit with other variable names.